### ISSUE TITLE
Please improve the mechanism for looking for a .mae file: install a templates.mae file as a default

### ISSUE THREAD

**Issue by yurivict** [schrodinger/coordgenlibs#64]
Here https://github.com/schrodinger/coordgenlibs/blob/master/sketcherMinimizer.cpp#L3520 a non-existent file user_templates.mae is encoded into the library at build time.

Suggested solution:

* install ```templates.mae``` into share/coordgen and default to it, otherwise allow the user to override this setting using some environment variable, ex. COORDGEN_TEMPLATES

**Comment by d-b-w** [comment]
Hi Yuri -
Thanks for this report/request.

I've been considering removing the user templates altogether, and adding a more robust mechanism for including them in the coordgen build. Most issues reported about coordgen have to do with linking maeparser or with run-time detection of the templates file. Additionally, removing this option would allow us to validate the templates at build time. It's all to easy to add a template that is never used.

Are you guys hoping for specific templates to be included?

**Comment by yurivict** [comment]
> Are you guys hoping for specific templates to be included?

I encountered this issue while updating the FreeBSD port and just installed the version included in the project.

**Comment by yurivict** [comment]
> Was there some sort of error that you experienced that led you to want a templates file?


Yes, OpenBabel-3.0 depends on coordgenlibs and in its cmake script is looking for templates.mae and failing to install find it because it was missing.

**Comment by d-b-w** [comment]
We could absolutely install a dummy templates.mae file. It just won't be used for anything.

**Comment by yurivict** [comment]
I'm just saying that the code references the .mae template in https://github.com/schrodinger/coordgenlibs/blob/master/sketcherMinimizer.cpp#L3520 that you compile into the library but don't install. People get confused, think that this file is required, and write cmake scripts to look for the file.

Hence, you should remove dangling file references from the library and document how templates are handled or used.

I am only an OpenBabel (and coordgenlibs) FreeBSD port maintainer and came across this issue when OpenBabel-3.0 failed to find coordgenlibs.

**Comment by d-b-w** [comment]
Thanks Yuri, and sorry for the confusion. Let me know if there's a better place to document than here: https://github.com/schrodinger/coordgenlibs#templates
